src/main/resources/application-schema.yml

```
@@ -0,0 +1,25 @@
+  jpa:
+    # ❗️가장 중요한 부분❗️
+    # DB를 직접 건드리지 않고, 스키마 생성 SQL을 파일로만 저장하게 함
+    properties:
+      jakarta:
+        persistence:
+          schema-generation:
+            create-source: metadata
+            # 생성된 SQL 스크립트를 어떤 파일에 저장할지 지정
+            schema-generation-create-script-source: META-INF/sql/create-schema.sql
+    hibernate:
+      ddl-auto: create # 'create'로 설정해야 스크립트가 생성됨
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)